main.tf

```diff
@@ -142,6 +142,12 @@ resource "aws_lambda_function" "this" {
     aws_iam_role_policy_attachment.vpc,
     aws_iam_role_policy_attachment.tracing,
   ]
+
+  lifecycle {
+    ignore_changes = [
```

Comment:
There are better solutions than this.

Timeouts should be rewritten with `dynamic`, like:

```hcl
  dynamic "timeouts" {
    for_each = length(var.timeouts) > 0 ? [true] : []

    content {
      create = try(var.timeouts.create, null)
      update = try(var.timeouts.update, null)
      delete = try(var.timeouts.delete, null)
    }
  }
```

Please update this PR and test it.

Reply:
Done